crates/api_common/src/utils.rs

```diff
@@ -615,6 +615,25 @@ pub async fn purge_image_posts_for_person(
   Ok(())
 }
 
+/// Delete a local_user's images
+pub async fn purge_local_user_images(
+  person_id: PersonId,
+  context: &LemmyContext,
+) -> Result<(), LemmyError> {
+  // Delete their images
+  if let Ok(local_user) = LocalUserView::read_person(&mut context.pool(), person_id).await {
+    let pictrs_uploads =
+      LocalImage::get_all_by_local_user_id(&mut context.pool(), &local_user.local_user.id).await?;
+
+    for upload in pictrs_uploads {
+      delete_image_from_pictrs(&upload.pictrs_alias, &upload.pictrs_delete_token, context)
```

Comment:
I believe you can use `purge_image_from_pictrs()` here and get rid of `delete_image_from_pictrs()`.